src/components/v5/frame/ColonyHome/partials/TokenBalance/hooks.ts

```diff
@@ -27,7 +27,11 @@ const calculateTotalFunds = async (
           isError = true;
         }
 
-        return (await total).add(new Decimal(balance).mul(currentPrice ?? 0));
+        const balanceInWeiToToken = new Decimal(balance).div(10 ** decimals);
```

Comment:
It's gonna be a nit-pick, it all works correct but since you're dividing by 10^decimals, the result is ETH rather than WEI. Would you mind updating the naming to reflect that?

Reply:
@jakubcolony oh I was under the impression that because this loop is going through your balances, I thought that you will also have non-ETH token balances in there with their own respective decimals as well which is why I wanted to call it something generic like "Token"

Reply:
@jakubcolony thanks for explaining that ETH could also mean the actual token unit, and is not just talking specifically about the Ethereum token! Changes pushed! 🚀